packages/react-native/android/build.gradle

```diff
@@ -53,6 +53,10 @@ android {
     }
   }
 
+  kotlinOptions {
+    freeCompilerArgs += ["-Xskip-metadata-version-check"]
+  }
```

Comment:
If we stick to this version we'll need to add this to the expo plugin to make sure that downstreams set this where appropriate